index.html

```diff
@@ -1,41 +1,74 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
 
 <head>
-    <meta charset="utf-8">
-    <meta name="description" content="Crooks Peak: It's a VR game studio">
-    <meta name="author" content="Crooks Peak">
-    <title>Crooks Peak</title>
-
-    <script src="rainBackground.js"></script>
-    <style>
-        .blinking{animation:blinkingText 4.8s infinite}@keyframes blinkingText{0%{color:#000}3%{color:#000}5%{color:transparent}99%{color:transparent}100%{color:#000}}
-    </style>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Crooks Peak | VR Game Studio</title>
+    <link rel="stylesheet" href="css/style.css" type='text/css'>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap" rel="stylesheet">
```

Comment:
As a performance hound and privacy advocate, I usually like to not use 3rd party hosted fonts, can you either have a local copy of the font (make sure to check font licensing for this) or use system fonts.